server/controller/userController.js

```diff
@@ -1,10 +1,10 @@
-const axios = require("axios");
+// const axios = require('axios');
 const bcrypt = require("bcryptjs");
 const jwt = require("jsonwebtoken");
```

Comment:
Does a web token show when you login? No tokens are showing for me when I test.

Reply:
If you haven't yet, please update your .env file, maybe that will fix this. I sent you the info on Slack.